.github/workflows/formatting.yaml

```diff
@@ -41,3 +54,34 @@ jobs:
       if: ${{ github.event_name == 'push' || github.event.pull_request.head.repo.full_name == github.repository }} 
       with:
         repo-token: ${{secrets.GITHUB_TOKEN}}
+
+  Build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2 
+    - name: Use Java 8
+      uses: actions/setup-java@v1
+      with:
+        java-version: "8"
+    - name: Use Node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: '12.x'
+    - name: Cache local npm dependencies
```

Comment:
This is getting intense for me here, haha. So the idea is, we only have to install node dependencies once between Linting above and Building/Testing here?

Reply:
Yes! and It makes it so that if any Action that sets up Maven or NodeJS has run before in any PR that this PR is built on and none of the dependencies have changed, GitHub won't re-build them. I read about it [here](https://docs.github.com/en/actions/configuring-and-managing-workflows/caching-dependencies-to-speed-up-workflows#about-caching-workflow-dependencies)
It cuts down IC check time drastically.